Packs/Zscaler/Integrations/Zscaler/Zscaler.yml

```
@@ -536,15 +452,53 @@ script:
       description: The number of unique URLs that are currently provisioned for your
         organization.
       type: Number
-  feed: false
-  isfetch: false
-  longRunning: false
-  longRunningPort: false
+  - arguments:
+    - description: Filer by user name
+      name: name
+    - description: Specifies the page offset.
+      name: page
+    - default: true
+      defaultValue: '100'
+      description: Specifies the page size.
+      name: pageSize
+    description: Get Zscaler users
+    name: zscaler-get-users
+  - arguments:
+    - description: The unique identifer for the user.
```

I think it will be easier to the user if you will write the structure of the identifier.
```suggestion
    - description: The unique identifer for the user. The structure of the identifier is: {"name":<name>, "email":<email>, "groups":<groups>, "department":<department>}. 
```